packages/istanbul-lib-source-maps/lib/get-mapping.js

```diff
@@ -98,7 +121,11 @@ function getMapping(sourceMap, generatedLocation, origFile) {
         return null;
     }
 
-    const start = sourceMap.originalPositionFor(generatedLocation.start);
+    const start = originalPositionTryBoth(
+      sourceMap,
+      generatedLocation.start.line,
+      generatedLocation.start.column
+    );
     let end = originalEndPositionFor(sourceMap, generatedLocation.end);
```

Comment:
I believe, in the case of my testing with `c8`, it was the `GREATEST_LOWER_BOUND` in the `originalEndPositionFor` that was resulting in `null` data; opting for `LEAST_UPPER_BOUND` works like a charm ... but I don't fully understand why both wouldn't return an element.